### PR TITLE
Readme about ovms usage in ams

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: extras/ams_wrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+OpenVINO Inference Server for LVA includes two components which require proper configuration.
+* OpenVINO Model Server - serves all models and executes inference operation
+* LVA REST API wrapper - translates LVA API, run pre and post processing operations, communicates with OVMS via localhost and gRPC interface.
+
+OpenVINO Model server requires file `/opt/ams_models/ovms_config.json` which is by default configured
+to use [4 exemplary models](../ams_models/ovms_config.json).
+
+LVA REST API wrapper requires model configuration files describing all enabled models.
+Models config files should be present in `/opt/ams_models/` folder and their name should have the name 
+matched with the model name configured in `ovms_config.json`.
```
Comment (mzegla) [inline extras/ams_wrapper/README.md:220]: They don't have to. Configuration files names could be arbitrary as long as they end with `_model.json`.

File: extras/ams_wrapper/README.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+               "value" : "blue",
+               "confidence" : 0.0417489744722843,
+               "name" : "color"
+            },
+            {
+               "name" : "color",
+               "value" : "black",
+               "confidence" : 0.0278004333376884
+            }
+         ]
```
Comment (mateusz-ciesielski) [inline extras/ams_wrapper/README.md:179]:
```suggestion
{
[
	{
		"confidence": 0.589816808700562, 
		"name": "color",
		 "value": "red"
	},
 	{
		"confidence": 0.289313167333603,
		 "name": "color",
		 "value": "white"
	},
 	{
		"confidence": 0.0435199774801731, 
		"name": "color", 
		"value": "gray"
	},
 	{
		"confidence": 0.0417489744722843, 
		"name": "color", 
		"value": "blue"
	},
 	{
		"confidence": 0.0278004333376884, 
		"name": "color", 
		"value": "black"
	},
 	{
		"confidence": 0.00414687860757113,
		 "name": "color", 
		"value": "green"
	},
 	{
		"confidence": 0.00365369999781251,
		 "name": "color",
		 "value": "yellow"
	}
]
```

Comment (mateusz-ciesielski) [reply extras/ams_wrapper/README.md:179]: Attributes are now sorted by confidence

File: extras/ams_wrapper/README.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+            {
+               "confidence" : 0.996138155460358,
+               "value" : "truck",
+               "name" : "type"
+            },
+            {
+               "name" : "type",
+               "confidence" : 0.000169166218256578,
+               "value" : "bus"
+            }
```
Comment (mateusz-ciesielski) [inline extras/ams_wrapper/README.md:202]:
```suggestion
	{
		"confidence": 0.996138155460358, 
		"name": "type",
		 "value": "truck"
	},
 	{
		"confidence": 0.00293471338227391,
		 "name": "type", 
		"value": "car"
	},
 	{
		"confidence": 0.000757944362703711,
		 "name": "type",
		 "value": "van"
	},
	{
		"confidence": 0.000169166218256578,
		 "name": "type", 
		"value": "bus"
	}
```

File: extras/ams_wrapper/README.md [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+         }
+      },
+    ],
+   "type" : "entity"
+}
+```
+
+### Classification models
+
+Classification models, in this category, give the results in softmax layer. This include a set
```
Comment (rasapala) [inline extras/ams_wrapper/README.md:124]: includes

File: extras/ams_wrapper/README.md [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+of probabilities in classes defined for the model.
+Each output of the model should have the shape `[1, C , ...]`. First dimension represent the batch size,
```
Comment (rasapala) [inline extras/ams_wrapper/README.md:126]: represents

File: extras/ams_wrapper/README.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+which should be set to 1. `C` represent all classes defined in the model. Remaining dimensions 
+are ignored (if present, first index is used).
+
+Examples of such models are available in the [OpenVINO Model Zoo](https://docs.openvinotoolkit.org/2020.2/_models_intel_index.html).
+In OVMS docker image for LVA is included the following classification model:
```
Comment (rasapala) [inline extras/ams_wrapper/README.md:131]: The following classification model is included in the OVMS docker image for LVA:

**Pull request merged**